coalib/parsing/FilterHelper.py

```diff
@@ -75,8 +75,11 @@ def apply_filters(filters, bears=None, sections=None):
         items = sections
         applier = _apply_section_filter
 
-    for filter in filters:
-        filter_name, *filter_args = filter
+    for filter_name, *filter_args in filters.items():
```

Comment:
In the original code filter_args gave a list instead of a single variable. (['syntax'] instead of syntax)
On using filter_args I got a single variable and *filter_args gave a list so I sticked with the original result because I didn't know if this change could give an error.

Reply:
`filters` is a list right?

Reply:
The original code takes filter as a list but according to my understanding the issue aims to change filters from a list to a dict as an argument. I might have misunderstood the issue.